src/app/lib/analyticsUtils/index.test.js

```diff
@@ -436,6 +436,29 @@ describe('getAtiUrl', () => {
     const data = [];
     expect(getAtiUrl(data)).toEqual('');
   });
+
+  it('should return ref as last item in url', () => {
```

Comment:
I think it should be ok that we create the url in the order of the parameters that we provide, but have a unit test where we are creating this url to make sure that ref is the last item?

Reply:
Added unit tests [here](https://github.com/bbc/simorgh/pull/6412/files/ec3beed20ad10458fb79a9ed576eb99b7e7fa584#diff-59330db613648920566256700c5d6c8d).